docker-compose-full-services-mac.yml

```diff
@@ -2,7 +2,7 @@ version: "2.1"
 services:
   geth:
     image: ethereum/client-go:v1.8.21
-    entrypoint: /bin/sh -c "apk add curl && geth --targetgaslimit "6200000" --dev --dev.period 1 --rpc --rpcapi personal,web3,eth,net --rpcaddr 0.0.0.0 --rpcvhosts=* --rpcport=8545"
+    entrypoint: /bin/sh -c "apk add curl && geth --targetgaslimit "6200000" --dev --dev.period 1 --rpc --rpcapi personal,web3,eth,net --rpcaddr 0.0.0.0 --rpcvhosts=* --rpcport=8545 --ws --wsorigins='*'"
```

Comment:
Are you sure about `*`, cannot we narrow the origins space?

Reply:
Please see [comment](https://github.com/ethereum/go-ethereum/issues/16608#issuecomment-386872941) to geth ws-origin issue. If I got it correctly this isn't user-agent scenario so web socket client don't include the `Origin` header to the connection. I found no option allowing to pass such header from our side.